www/pages/user/[id]/index.js

```diff
@@ -21,7 +21,7 @@ const User = () => {
     }
   }, [ user ] );
 
-  const loaded = ownedMeetings.length || participantMeetings.length;
+  const loaded = ownedMeetings.length >= 0 || participantMeetings.length >= 0;
```

Comment:
Do you think we should wait for all meetings to load before saying that it is loaded? Also can you add a todo to notify when a fetch fails.

Reply:
yeah but wont it return the entire batch, not one at a time?

Reply:
Yep, sounds good, we should probably have the `useEffect` rely on a `loading` var though and then set `loading = false` at the end the nested async function. We will still need to handle errors there anyway.